src/background/contextmenus.js

```diff
@@ -42,6 +42,7 @@ export class ContextMenus extends Component {
             this.sendMessage('shareURL', info.linkUrl);
             break;
         }
+        browser.browserAction.openPopup();
```

Comment:
I think I found a better solution:
- let's remove the 'await' as you did in the first version of this PR
- let's call: `return Promise.all(returnValues);` instead of `return returnValues;`

We still need to keep some of this PR to fix the message "conflicts".